etl/scripts-ccao-data-warehouse-us-east-1/ccao/ccao-commercial_valuation.R

```diff
@@ -30,9 +30,15 @@ char_cols <- c(
   "f/r",
   "carwash",
   "ceilingheight",
-  "2023permit/partial/demovalue",
+  "permit/partial/demovalue",
```

Comment:
These columns don't need to be specific to any given year. It was fine in the past, but we'd have to keep adding a new one every year,